.github/workflows/release-binaries.yaml

```diff
@@ -0,0 +1,29 @@
+name: Release binaries
+
+permissions:
+  contents: write
+
+on:
+  workflow_dispatch:
+  release:
+    types: [ created ]
```

Comment:
Would it be worth also uploading binaries on tagging? I'm not familiar with the release process. 

```
on:
  push:
    tags:
      - v[0-9]+.*
```

Reply:
Never mind, I see the release can create a tag and it's just better because you have to describe what you did.